sympy/stats/tests/test_joint_rv.py

```diff
@@ -19,7 +19,7 @@ def test_Normal():
     assert marginal_distribution(n, 0, 1)(1, 2) == 1/(2*pi)
     assert integrate(density(m)(x, y), (x, -oo, oo), (y, -oo, oo)).evalf() == 1
     N1 = Normal('N1', [1, 2], [[x, 0], [0, y]])
-    assert str(density(N1)(0, 0)) == "exp(-(4*x + y)/(2*x*y))/(2*pi*sqrt(x*y))"
+    assert density(N1)(0, 0) == exp(-2/y - 1/(2*x))/(2*pi*sqrt(x*y))
```

Comment:
Not sure why this was a str assert before. Is there a reason to change this?

Reply:
Sometimes str tests like this are there because the resulting function is in an unevaluated form. In this case, it looks like `exp(-(4*x + y)/(2*x*y))/(2*pi*sqrt(x*y))` doesn't create `exp(-(4*x + y)/(2*x*y))/(2*pi*sqrt(x*y))` because the minus sign gets distributed. So if you wrote `assert density(N1)(0, 0)) == exp(-(4*x + y)/(2*x*y))/(2*pi*sqrt(x*y))` the test would fail even though `str(density(N1)(0, 0)) == "exp(-(4*x + y)/(2*x*y))/(2*pi*sqrt(x*y))"`.

Actually you can avoid the `str` thing here by using `exp((4*x + y)/(-2*x*y))/(2*pi*sqrt(x*y))`) instead.

If the result is expanded now, then it isn't relevant any more.

Reply:
As to how this comes about from this change, the density of a multivariate normal is computed via a matrix exponential, which computes the eigenvalues.